buyer-app/base-app/src/state/ontologies/selectors.js

```diff
@@ -102,6 +102,28 @@ const audienceOntology = {
           label: "$100,001+"
         }
       ]
+    },
```

Comment:
Should we remove other filters to avoid confusion?

Reply:
Not yet, I think. They're using this as it is, ignoring the audience field. I would keep it, and just let them know that we're actually filtering just by country (all other fields are ignored).